uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -138,29 +150,40 @@ define([
     }
 
     /**
-     * @TODO This function utilises external resource
-     * vm.absenceTypes - this sould be refactored
-     *
      * Checks if the given leave request can be cancelled
      *
-     * Allow request cancellation values refer to the following constants:
-     * 1 = REQUEST_CANCELATION_NO
-     * 2 = REQUEST_CANCELATION_ALWAYS
-     * 3 = REQUEST_CANCELATION_IN_ADVANCE_OF_START_DATE
+     * @TODO This function utilises external resource
+     * vm.absenceTypes - this sould be refactored
      *
      * @return {Boolean}
      */
-    function canLeaveRequestBeCancelled () {
-      var allowCancellationValue = vm.absenceTypes[vm.leaveRequest.type_id].allow_request_cancelation;
+    function canLeaveRequestBeCancelled (leaveRequestStatus) {
+      var allowCancellationValue = vm.absenceTypes[vm.leaveRequest.type_id]
```

Comment:
Please keep the `.` in the original(previous) line. Readability is better that way.

Reply:
Done.